hr_timesheet_sheet_attendance/tests/test_hr_attendance.py

```diff
@@ -8,7 +8,7 @@
 class TestHrAttendance(HrTimesheetTestCases):
     def setUp(self):
         super(TestHrAttendance, self).setUp()
-        checkInDate = datetime.datetime(2018, 12, 12, 10, 0, 0)
+        checkInDate = datetime.datetime(2022, 2, 12, 10, 0, 0)
```

Comment:
These changes are unnecessary. Dates always be on the past overtime. For reliable dates, use `freezegun` if needed.